product_categories_in_sales/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2020 Manuel Regidor <manuel.regidor@sygel.es>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Product Categories in Sales",
+    "summary": "Shows 'Product Categories' menu item in Sales",
+    "version": "12.0.1.0.1",
+    "category": "Product",
+    "website": "https://www.sygel.es",
+    "author": "Sygel",
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "depends": [
+        "sale",
+        "product",
```

Comment:
I think that the second dependency (product) is unnecessary.

Reply:
Yes, you're right. Thank you.